### PR TITLE
Microvers Pull Request for index.html updates

### PR DESCRIPTION
**Pull request by Hamayounali**
Updated Hamayoun project. 
Added Body section and about section in Index.html

Note: Accidentally the updates are curried out on the main branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,5 +88,183 @@
 	  	
 	</div>
 </section>
+
+<section id="body">
+	<div class="flex-body">
+		<div class="recent-work">
+			My Recent Works
+			<hr>
+		</div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the title (e.g., "My Recent Works") uses a header tag `hx`.

File: assets/css/style.css [hunk 2]
```diff
@@ -88,3 +88,236 @@
...
+  height: 3455px;
+  width: 375px;
+  top: 682px;
+}
+
+.flex-body > div {
+  padding: 23px;
+  color: black;
+  flex-grow: 1;
+}
```
Comment (ShahierNashaat) [inline assets/css/style.css]: - [x] Kindly be noticed that the content of the project is aligned with the flexbox.

File: assets/css/style.css [hunk 3]
```diff
@@ -88,3 +88,236 @@
...
+
+/* Body of the page */
+.flex-body {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  height: 3455px;
+  width: 375px;
+  top: 682px;
+}
```
Comment (ShahierNashaat) [inline assets/css/style.css]: - [x] Kindly be noticed that the main container of the projects has a `display:grid`.

File: index.html [hunk 4]
```diff
@@ -72,5 +88,183 @@
...
+		
+		<div class="stories">
+			<img src="assets/images/placeholder-1.png" alt="stories">
+			<div class="work-sample-2"> Multy-Post Stories </div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the project title (e.g., "Tonic") uses a header tag `hx`.

File: index.html [hunk 5]
```diff
@@ -72,5 +88,183 @@
...
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see-2">See Project</div>
+		</div>
+
+		<!-- Start of Cards -->
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the project title (e.g., "Tonic") uses a header tag `hx`.

File: index.html [hunk 6]
```diff
@@ -72,5 +88,183 @@
...
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the project title (e.g., "Tonic") uses a header tag `hx`.

File: index.html [hunk 7]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the project title (e.g., "Tonic") uses a header tag `hx`.

File: index.html [hunk 8]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the project title (e.g., "Tonic") uses a header tag `hx`.

File: index.html [hunk 9]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the project title (e.g., "Tonic") uses a header tag `hx`.

File: index.html [hunk 10]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the project title (e.g., "Tonic") uses a header tag `hx`.

File: index.html [hunk 11]
```diff
@@ -72,5 +88,183 @@
...
+			<div class="work-sample-detail">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+			</div>
+			<div class="tags-2">
+				<div class="tag">css</div>
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see-2">See Project</div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

File: index.html [hunk 12]
```diff
@@ -72,5 +88,183 @@
...
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

File: index.html [hunk 13]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

File: index.html [hunk 14]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

File: index.html [hunk 15]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

File: index.html [hunk 16]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

File: index.html [hunk 17]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

File: index.html [hunk 18]
```diff
@@ -72,5 +88,183 @@
...
+			<div class="work-sample-2"> Multy-Post Stories </div>
+			<div class="work-sample-detail">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+			</div>
+			<div class="tags-2">
+				<div class="tag">css</div>
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

File: index.html [hunk 19]
```diff
@@ -72,5 +88,183 @@
...
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

File: index.html [hunk 20]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

File: index.html [hunk 21]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

File: index.html [hunk 22]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

File: index.html [hunk 23]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

File: index.html [hunk 24]
```diff
@@ -72,5 +88,183 @@
...
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

File: index.html [hunk 25]
```diff
@@ -72,5 +88,183 @@
...
+	</div>
+</section>
+<br> <br> <br>
+<section class="head">
+	<!-- Heading Section  -->
+	<div class="flex-about">
+	  <div class="about-me">
+	  	<div class="head-title"> 
+	    	About Me 
+	    </div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the title (e.g., "About myself") uses a header tag `hx`.

File: index.html [hunk 26]
```diff
@@ -72,5 +88,183 @@
...
+	  	<div class="head-body">
+	  		Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+	  	</div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the paragraph (e.g., "I'm a software ...") is a `p` tag.

File: index.html [hunk 27]
```diff
@@ -72,5 +88,183 @@
...
+	  	<div class="see-3">
+	  		Get My Resume
+	  	</div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

File: index.html [hunk 28]
```diff
@@ -72,5 +88,183 @@
...
+	  </div>
+	  <div>
+	  	<span class="lang-title"> <img src="assets/images/rectangle.png" alt="rectangle"> Skills </span> 
+	  	<div class="lang">
+		  	<p>CodeKit</p>
+		  	<p>GitHub</p>
+		  	<p>Code Pen</p>
+		  	<p>GitLab</p>
+		  	<p>Terminal</p>
+	    </div>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Kindly be noticed that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`.
- [x] And the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Hamayounali and @hamayounali5 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [Optional] Kindly remove the top part of your design as it just shows how it looks on mobile and not to design it.
Screenshot from 2022-05-28 14-50-29

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css (hunk 3 above)
Comment (Hamayounali) [reply assets/css/style.css]: Thank you, I have changed it.

File: assets/css/style.css (hunk 2 above)
Comment (Hamayounali) [reply assets/css/style.css]: The content of the project is inside of grid

File: index.html [hunk 29]
```diff
@@ -72,5 +88,177 @@
 	  	
 	</div>
 </section>
+
+<section id="body">
+	<div class="grid-body">
+		<h1 class="recent-work">
+			My Recent Works
+		</h1>
+		<div class="stories">
```
Comment (m05e5) [inline index.html:97]: -  _[optional]_ Please I strongly recommend changing this and the others to `h2`, `h3`, or any other `hx`. As part of the best practices, its recommended to have only one `h1` tag in your file

**Review by m05e5 — APPROVED** [review]
# Hi @Hamayounali , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**